### DISCUSSION TITLE
New footer implementation

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#20135] [Ideas] [answered]
![sss](https://user-images.githubusercontent.com/74463310/188348659-bec8a888-4dbe-4a3c-bf0e-8dc6f69b2d3c.png)
![s](https://user-images.githubusercontent.com/74463310/188348669-061d2468-e260-43d7-8bbb-4306d7c1b8f9.png)
mostly done as can be seen, 

i filled in the fonts with similar ones since the exact ones were unavailable, can i somehow access them?

**Comment by smoogipoo** [comment]
@arflyte Where can the icons in these buttons be found?

**Reply by Joehuu** [reply]
Those are from font awesome:
- https://fontawesome.com/icons/right-left?s=solid&f=classic
- https://fontawesome.com/icons/shuffle?s=solid&f=classic
- https://fontawesome.com/icons/gear?s=solid&f=classic

The gear used in figma is "regular" and limited to pro so I linked the "solid" one.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/20148
